apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -118,10 +118,11 @@ type WorkspaceConfig struct {
 	// "15m", "20s", "1h30m", etc. If not specified, the default value of "5m" is used.
 	ProgressTimeout string `json:"progressTimeout,omitempty"`
 	// IgnoredUnrecoverableEvents defines a list of Kubernetes event names that should
-	// be ignored when deciding to fail a DevWorkspace startup. This option should be used
-	// if a transient cluster issue is triggering false-positives (for example, if
-	// the cluster occasionally encounters FailedScheduling events). Events listed
-	// here will not trigger DevWorkspace failures.
+	// be ignored when deciding to fail a DevWorkspace startup.
```

Comment:
I'm not entirely sure we need to mention the cluster auto-scaler in DWO (or rewrite the docs here). It might be better to mention this in the Che Cluster CRD documentation, since the ignoredUnrecoverableEvents [can be configured](https://github.com/eclipse-che/che-operator/pull/1864) from the Che Cluster CRD. 

Instead, I would suggest:
- Mentioning "By default, the FailedScheduling is ignored"
- Removing the `"(for example, if the cluster occasionally encounters FailedScheduling events)"` since this example is no longer valid now that the FailedScheduling event is ignored by default